contracts/manager/Actor.sol

```diff
@@ -6,43 +6,43 @@ pragma solidity ^0.8.20;
 // import {console} from "hardhat/console.sol";
```

Comment:
Needs removing really before the audit.

Reply:
It is not needed, I think; it is commented. But it is quite easy to remove all of them.